.circleci/config.yml

```diff
@@ -47,27 +47,6 @@ jobs:
             --buffer-size 8192 \
             http://localhost:8080/
 
-  # Build and push 'latest' Docker tag.
```

Comment:
We shouldn't remove this until we're fully on github actions. Is there a problem leaving it until we deprecate the Docker hub images?

Reply:
Sure, I can make this a separate PR.